src/parser/parse_error.ml

```diff
@@ -81,3 +81,8 @@ let truncated_code_block_meta : Loc.span -> Warning.t =
 
 let truncated_code_block : Loc.span -> Warning.t =
   Warning.make ~suggestion:"add ']}'." "Missing end of code block."
+
+let end_not_allowed : in_what:string -> Loc.span -> Warning.t =
+ fun ~in_what ->
+  Warning.make ~suggestion:"add '}'." "End of text is not allowed in %s."
```

Comment:
I've found a way to avoid the redundant `~suggestion` argument. What do you think ?